### PR TITLE
Create helper to return directly target of link resources

### PR DESCRIPTION
**Pull request by tgalopin**
Fix https://github.com/puli/issues/issues/126.

### CODE REVIEW

File: src/Resource/LinkResource.php [hunk 1]
```diff
@@ -52,6 +52,18 @@ public function getTargetPath()
     /**
      * {@inheritdoc}
      */
+    public function getTarget()
+    {
+        if (!$this->getRepository()) {
+            throw ResourceNotFoundException::forPath($this->getTargetPath());
```
Comment (webmozart) [inline src/Resource/LinkResource.php]: IMO this exception is misleading. The error is not that the target resource is not found, but that the link resource is not attached to a repository.

I think you should throw a RuntimeException here (and in similar cases).

Comment (webmozart) [inline src/Resource/LinkResource.php]: Also, the test for this is missing.

**Comment by tgalopin** [comment]
I made the changes

File: tests/AbstractEditableRepositoryTest.php [hunk 2]
```diff
@@ -473,4 +480,13 @@ public function testDirectoryLink()
         $this->assertSame($this->readRepo, $target->getRepository());
         $this->assertCount(2, $target->listChildren());
     }
+
+    /**
+     * @expectedException RuntimeException
```
Comment (webmozart) [inline tests/AbstractEditableRepositoryTest.php]: Can you adapt the type of the expected Exception?

**Comment by webmozart** [comment]
Thanks @tgalopin! :)

**Pull request merged**